### PR TITLE
Rewrite lookup table files to improve readability

### PR DESCRIPTION
**Pull request by lukey-luke**
## What was the problem?

 - This PR rewrites the lookup table for the Contention Classification to use a single .csv file per request of product management.
 - This PR updates the lookup tables classes in lookup_table.py to return a dict containing keys `"classification_code"` and `"classification_name"` in order to get access to both pieces of information at the same time

Associated tickets or Slack threads:
- https://github.com/department-of-veterans-affairs/vagov-claim-classification/issues/476

## How does this fix it?[^1]

 - Design provided a new format for a .csv file. This PR uses ContentionTextLookupTable to parse the new .csv
 - 

## How to test this PR
- manual testing of `/classifier` and `/vagov-claim-classifier`
- run `pytest`

[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).

### CODE REVIEW

File: domain-cc/cc-app/src/python_src/util/lookup_table.py [hunk 1]
```diff
@@ -144,6 +151,36 @@ def get_v1_lookup_table(filepath: str, input_key: str, output_key: str) -> dict:
     return classification_code_mappings
 
 
+class ContentionTextLookupTable:
```
Comment (lukey-luke) [inline domain-cc/cc-app/src/python_src/util/lookup_table.py:98]: add class-level docstring explaining what this class is for

File: domain-cc/cc-app/src/python_src/util/lookup_table.py [hunk 2]
```diff
@@ -144,6 +151,36 @@ def get_v1_lookup_table(filepath: str, input_key: str, output_key: str) -> dict:
...
+    CSV_FILEPATH = os.path.join(
+        os.path.dirname(__file__),
+        "data",
+        "condition_dropdown_lookup_table",
+        contention_lut_csv_filename,
+    )
+    input_key = "CONTENTION TEXT"
+    output_key = "CLASSIFICATION CODE"
```
Comment (lukey-luke) [inline domain-cc/cc-app/src/python_src/util/lookup_table.py:111]: capitalize variables to indicate constants

File: domain-cc/cc-app/src/python_src/util/lookup_table.py [hunk 3]
```diff
@@ -144,6 +151,36 @@ def get_v1_lookup_table(filepath: str, input_key: str, output_key: str) -> dict:
...
+            for row in csv_reader:
+                table_key = row[self.input_key].strip().lower()
+                self.classification_code_mappings[table_key] = int(row[self.output_key])
+
+    def get(self, input_str: str, fallback=None):
+        input_str = input_str.strip().lower()
+        classification = self.classification_code_mappings.get(input_str, fallback)
+        print(f"(ContentionTextLookupTable) returning {classification} for {input_str}")
+        # return self.classification_code_mappings.get(input_str, fallback)
+        return classification
```
Comment (lukey-luke) [inline domain-cc/cc-app/src/python_src/util/lookup_table.py:128]: improve readability by returning 
type Classification or a named tuple or _something_ that includes both `classification_code` and `classification_name`

**Comment by tyler-spangler6** [comment]
Should we update the name for the autosuggestion csv file as well? In the data folder it is named with the PACT act release, but in the new master spreadsheet it is named as "Suggested Conditions v[ ]". This also matches Eva's comment in the zenhub ticket regarding file names.

File: domain-cc/cc-app/src/python_src/util/lookup_table.py [hunk 4]
```diff
@@ -144,6 +95,43 @@ def get_v1_lookup_table(filepath: str, input_key: str, output_key: str) -> dict:
...
+                table_key = row[self.input_key].strip().lower()
+                self.classification_code_mappings[table_key] = {
+                    "classification_code": int(row[self.output_key]),
+                    "classification_name": row["CLASSIFICATION TEXT"],
+                }
+
+    def get(self, input_str: str, default_value=LUT_DEFAULT_VALUE):
+        input_str = input_str.strip().lower()
+        classification = self.classification_code_mappings.get(input_str, default_value)
+        print(f"(ContentionTextLookupTable) returning {classification} for {input_str}")
```
Comment (tyler-spangler6) [inline domain-cc/cc-app/src/python_src/util/lookup_table.py]: Should we remove this print statement?

Comment (lukey-luke) [reply domain-cc/cc-app/src/python_src/util/lookup_table.py]: done

**Comment by lukey-luke** [comment]
> Should we update the name for the autosuggestion csv file as well? In the data folder it is named with the PACT act release, but in the new master spreadsheet it is named as "Suggested Conditions v[ ]". This also matches Eva's comment in the zenhub ticket regarding file names.

@tyler-spangler6  I don't recall this from the synchronous conversation we had prior, but I found [[Release notes] Auto-suggestions release notes](https://docs.google.com/spreadsheets/d/1-mwSyrs4gDtZ9ouJ3Qu68_aWHuIgHMN-ZzEr8h8U_jU/edit?gid=1342210777#gid=1342210777) on google drive and added the `Suggested Conditions v1.1 Flat` tab to abd-vro

**Review by tyler-spangler6 — APPROVED** [review]
LGTM

**Pull request merged**